pyproject.toml

```diff
@@ -156,6 +156,7 @@ tabulate = ">=0.8.9,<0.10.0"
 jira = "^3.5.2"
 junitparser = "^3.1.0"
 tenacity = "^8.2.3"
+pygraphviz = "*"
```

Comment:
is this dependency required for CI to run, or just locally?

Reply:
It required for CI